python/src/opendp/extras/numpy/_make_np_sscp/__init__.py

```diff
@@ -1,52 +1,38 @@
 from opendp.extras._utilities import to_then
 from opendp.extras.numpy import _sscp_domain
 from opendp.mod import Domain, Metric, Transformation
-from opendp._lib import import_optional_dependency
 from opendp._internal import _make_transformation
 
 # planning to make this public, but may make more API changes
 
 
-def make_np_sscp(
-    input_domain: Domain, input_metric: Metric, output_metric: Metric
-) -> Transformation:
+def make_np_sscp(input_domain: Domain, input_metric: Metric) -> Transformation:
     """Construct a Transformation that computes a covariance matrix from the input data.
 
     :param input_domain: instance of `array2_domain(size=_, num_columns=_)`
     :param input_metric: instance of `symmetric_distance()`
-    :param output_metric: either `symmetric_distance()` or `l2_distance()`
 
     :return: a Measurement that computes the DP sum
     """
     import opendp.prelude as dp
-    np = import_optional_dependency('numpy')
 
     dp.assert_features("contrib", "floating-point")
 
     if not str(input_domain).startswith("NPArray2Domain"):
-        raise ValueError("input_domain must be a 2d-numpy array domain")  # pragma: no cover
+        raise ValueError(
+            "input_domain must be a 2d-numpy array domain"
+        )  # pragma: no cover
 
+    if input_metric != dp.symmetric_distance():
+        raise ValueError("input metric must be symmetric distance")  # pragma: no cover
```

Comment:
For here and below: Include the `input_domain` or `input_metric` in the error message?

Test coverage?